src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx

```diff
@@ -2,18 +2,23 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import {Client} from 'app/api';
-import {Environment, Organization} from 'app/types';
+import {Environment, Organization, SelectValue} from 'app/types';
 import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
 import {addErrorMessage} from 'app/actionCreators/indicator';
 import {defined} from 'app/utils';
 import {getDisplayName} from 'app/utils/environment';
 import {t, tct} from 'app/locale';
+import Feature from 'app/components/acl/feature';
 import FormField from 'app/views/settings/components/forms/formField';
 import SearchBar from 'app/views/events/searchBar';
 import SelectField from 'app/views/settings/components/forms/selectField';
 import space from 'app/styles/space';
 import theme from 'app/utils/theme';
 import Tooltip from 'app/components/tooltip';
+import {Column, AGGREGATIONS, FIELDS, TRACING_FIELDS} from 'app/utils/discover/fields';
+import Field from 'app/views/settings/components/forms/field';
+import {ColumnEditRow} from 'app/views/eventsV2/table/columnEditRow';
+import {FieldValue, FieldValueKind} from 'app/views/eventsV2/table/types';
```

Comment:
Once you have the column editor integrated into alerts I'll move all this code into a shared component directory. Cross view imports always make me sad.